### PR TITLE
doc: improve pending release notes and CephFS

### PR DESCRIPTION
**Pull request by anthonyeleven**
doc: improve pending release notes and CephFS

Signed-off-by: Anthony D'Atri <anthonyeleven@users.noreply.github.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -74,38 +74,36 @@
...
+  latency and lifecycle processing failures. To check whether a versioned
   bucket has unnecessary olh entries, users can now run ``radosgw-admin
   bucket check olh``. If the ``--fix`` flag is used, the extra entries will
-  be safely removed. A distinct issue from the one described thus far, it is
-  also possible that some versioned buckets are maintaining extra unlinked
-  objects that are not listable from the S3/ Swift APIs. These extra objects
-  are typically a result of PUT requests that exited abnormally, in the middle
-  of a bucket index transaction - so the client would not have received a
+  be safely removed. An additional issue  is that some versioned buckets are
+  may maintain extra unlinked
```
Comment (zdover23) [inline PendingReleaseNotes:99]: There's one too many verbs in this sentence. "An additional issue is that some versioned buckets are may maintained extra unlimited objects that are not listable via the S3/ Swift APIs." should probably be "An additional issue is that some versioned buckets may maintain extra unlinked objects that are not listable via the S3/Swift APIs."

File: PendingReleaseNotes [hunk 2]
```diff
@@ -74,38 +74,36 @@
...
+  objects that are not listable via the S3/ Swift APIs. These extra objects
+  are typically a result of PUT requests that exited abnormally in the middle
+  of a bucket index transaction, and thus the client would not have received a
   successful response. Bugs in prior releases made these unlinked objects easy
-  to reproduce with any PUT request that was made on a bucket that was actively
-  resharding. Besides the extra space that these hidden, unlinked objects
-  consume, there can be another side effect in certain scenarios, caused by
-  the nature of the failure mode that produced them, where a client of a bucket
+  to reproduce with any PUT request made on a bucket that was actively
+  resharding. In addition to the extra space that these hidden, unlinked objects
```
Comment (zdover23) [inline PendingReleaseNotes:105]: It seems to me that we can dispense with "In addition to the extra space that these hidden, unlinked objects consume,". It seems to me that the position of this sentence in the text immediately after the discussion of the cost of hidden, unlinked objects already implies that the information about the object's inconsistent state is provided in addition to what has come before.

File: PendingReleaseNotes [hunk 3]
```diff
@@ -118,16 +116,16 @@
   recommend that users with versioned buckets, especially those that existed
   on prior releases, use these new tools to check whether their buckets are
   affected and to clean them up accordingly.
-* rgw: The User Accounts feature unlocks several new AWS-compatible IAM APIs
-  for the self-service management of users, keys, groups, roles, policy and
+* RGW: The user accounts feature unlocks several new AWS-compatible IAM APIs
```
Comment (zdover23) [inline PendingReleaseNotes:119]: What do you think of this?  `The "user accounts" feature` ?

That keeps this complex adjective unambiguous in the mind of the first-time reader.

Let me know what you think.

**Review by zdover23 — APPROVED** [review]

File: PendingReleaseNotes (hunk 3 above)
Comment (anthonyeleven) [reply PendingReleaseNotes]: Points taken and revisions pushed

**Comment by zdover23** [comment]
I've squashed and rebased this (and removed the merge of "main" into `improve-mgr-disable`. Everything seems to be working correctly. When this has passed its Jenkins tests, I'll merge and backport this PR.

**Comment by anthonyeleven** [comment]
Ack.  I didn't quite understand where the conflict arose, and I attempted to resolve via the web interface :-/

**Comment by zdover23** [comment]
> Ack. I didn't quite understand where the conflict arose, and I attempted to resolve via the web interface :-/

That is the correct emoji in response to the web interface.

**Comment by zdover23** [comment]
jenkins test make check

**Comment by zdover23** [comment]
jenkins test make check

File: doc/cephfs/fs-volumes.rst [hunk 4]
```diff
@@ -1422,18 +1422,18 @@ set with this id was present in the database
 
 .. _disabling-volumes-plugin:
 
-Disabling Volumes Plugin
-------------------------
+Disabling the Volumes Plugin
```
Comment (batrick) [inline doc/cephfs/fs-volumes.rst:1425]: separate pr as well

**Review by batrick — CHANGES_REQUESTED** [review]

**Comment by anthonyeleven** [comment]
@zdover23

**Comment by zdover23** [comment]
> @zdover23

I'm already working on it.

**Comment by anthonyeleven** [comment]
Thanks.  I painted myself into a corner here and had to re-clone my local checkout.  LMK if I should just abandon and recast.

**Comment by zdover23** [comment]
> Thanks. I painted myself into a corner here and had to re-clone my local checkout. LMK if I should just abandon and recast, as your git-fu is much stronger than mine.

I'll sort this out. You don't have to abandon this PR. I will be cutting it down, however, and raising separate pull requests for each of the changes. In general, make docs changes only to one file per PR (unless you are creating a link to a section from another file, in which case changes to two files is fine).

Even though this grab-bag PR has gotten tangled in the make check it has pointed up at least two places where the documentation in the main branch has diverged from the documentation in the release branches. In those cases, documentation changes were included in PRs that made code changes and they were never properly backported. So: even though this is a costly untangling, it is a useful one.

**Comment by anthonyeleven** [comment]
In the words of Yngwie Malmsteen's road manager, you sir are a gentleman and a scholar.

**Comment by zdover23** [comment]
> In the words of Yngwie Malmsteen's road manager, you sir are a gentleman and a scholar.

Well, thanks.

This is going to take a while, because several changes will have to be made one after another in order to bring the release branches into agreement with the main branch, and each of those changes is dependent upon our waiting for the Jenkins CI mechanism to complete its business. So expect to have to approve at least nine more pull requests today.

**Comment by zdover23** [comment]
jenkins test make check

**Comment by zdover23** [comment]
jenkins test windows

File: doc/cephfs/fs-volumes.rst (hunk 4 above)
Comment (zdover23) [reply doc/cephfs/fs-volumes.rst]: I have moved this into a separate PR here: https://github.com/ceph/ceph/pull/60465.

**Review by vshankar — APPROVED** [review]
LGTM (cephfs bits)

**Review by batrick — APPROVED** [review]

**Comment by zdover23** [comment]
No backports necessary for updates to PendingReleaseNotes.

**Pull request merged**